ksqldb-functional-tests/src/test/resources/query-validation-tests/protobuf.json

```diff
@@ -79,6 +79,51 @@
       "inputs": [{"topic": "input", "value": {"i": 1, "l": 1, "d": 1.2, "b": true, "s": "foo"}}],
       "outputs": [{"topic": "OUTPUT", "value": {"I": 1, "L": 1, "D": 1.2, "B": true, "S": "foo"}}]
     },
+    {
+      "name": "protobuf primitives with null",
+      "statements": [
+        "CREATE STREAM INPUT (K STRING KEY, i INTEGER, l BIGINT, d DOUBLE, b BOOLEAN, s STRING) WITH (kafka_topic='input', value_format='PROTOBUF');",
+        "CREATE STREAM OUTPUT as SELECT * FROM input;"
+      ],
+      "inputs": [
+        {"topic": "input", "value": {"i": null, "l": 0, "d": null, "b": false, "s": null}},
+        {"topic": "input", "value": {"i": 0, "l": null, "d": 0.0, "b": null, "s": ""}}
+      ],
+      "outputs": [
+        {"topic": "OUTPUT", "value": {"i": 0, "l": 0, "d": 0.0, "b": false, "s": ""}},
+        {"topic": "OUTPUT", "value": {"i": 0, "l": 0, "d": 0.0, "b": false, "s": ""}}
+      ]
+    },
+    {
+      "name": "protobuf primitives with null and nullable as optional",
```

Comment:
I think QTT has something set up for validating that the schemas produced by the statements are as expected. That could be insightful to add to these tests, since otherwise the behavior of this test and the one below appear identical.

It's not the biggest deal if it's not added, though, since the historic plan spec files (once generated) should include schema validation. Adding it would be a way to make it more explicit that this is the intention behind the test, though.

Reply:
Done